ios/test_app.rb

```diff
@@ -142,15 +144,24 @@ def make_project!(xcodeproj, project_root, target_platform)
   FileUtils.cp(File.join(src_xcodeproj, 'project.pbxproj'), dst_xcodeproj)
   FileUtils.ln_sf(File.join(src_xcodeproj, 'xcshareddata'), dst_xcodeproj)
 
+  # Link Xcode build configuration files
```

Comment:
If we no longer need to modify them, we can just move them into `ReactTestApp` folder instead. `ReactTestApp` gets linked below and we'll no longer need this block.

Reply:
Ah, even better 👍